view_conversations/conv-viewer-webapi/src/routes.ts

```diff
@@ -64,20 +48,18 @@ export function setup(
 
       // TODO remove outer try wrapper unless it get used.
       const sqlQuery = `SELECT *
-      FROM ${table}
-      WHERE conversation_id="${conv_id}"
-      LIMIT 100`;
+             FROM ${table}
+             WHERE conversation_id="${conv_id}"
+             LIMIT 100`;
       // Query options list:
-      // https://cloud.google.com/bigquery/docs/reference/v2/jobs/query
-      const options = {
-        query: sqlQuery,
-        useLegacySql: false,  // Use standard SQL syntax for queries.
+      // https://cloud.google.com/spanner/docs/getting-started/nodejs/#query_data_using_sql
+      const query: spanner.Query = {
+        sql: sqlQuery
       };
 
-      await bqClient.query(options).then(results => {
+      await spannerDatabase.run(query).then(results => {
         const rows = results[0];
-        convertToSimpleTypes(rows);
-        res.status(httpcodes.OK).send(JSON.stringify(rows, null, 2));
+        res.status(httpcodes.OK).send(db_types.parseOutputRows<db_types.OutputRow[]>(rows));
```

Comment:
Are you sure you can send without wrapping with JSON.stringify()? (I'm not actually sure what the answer is, I just noticed that it was there before and it was removed, so I want to make sure you tested that this still works to send an object here).

Reply:
I wrapped this with JSON.stringfy(). I have a question here to @iislucas though, when this is called from conv-viewer-webapp, the result will be JSON.stringfy() again, is that part necessary?